src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1298,6 +1312,10 @@ private void InitEditor()
 
     private void InitEditorFresh()
     {
+        targetPatch = null;
+
+        playerPatchOnEntry = CurrentGame.GameWorld.Map.CurrentPatch;
```

Comment:
Why where these two moved here?

Reply:
Those were moved because `UpdateReportTabStatistics` needs the current patch in `OnEditorReady` so `platerPatchOnEntry` needs to be immediately assigned there.

Reply:
I think it wouldn't hurt to tweak their positioning nonetheless, also probably makes sense to set the patch asap so things could run correctly down the line.

Reply:
Wouldn't it also make sense to move the other bits, like reseting the symmetry etc. here? Or do those need to be where they are due to save loading stuff (I don't have the full code on hand right now to check that)?

Reply:
They could be moved too but I don't think it would make any significant impact since `OnEditorEntry` doesn't depend on them. However for consistency it'd probably be nice to also move them.